firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1908,6 +1893,41 @@ class BrowserViewController: UIViewController,
         }
     }
 
+    // MARK: Overlay View
+    // Disable search suggests view only if user is in private mode and setting is enabled
+    private var shouldDisableSearchSuggestsForPrivateMode: Bool {
+        let featureFlagEnabled = featureFlags.isFeatureEnabled(.feltPrivacySimplifiedUI, checking: .buildOnly)
+        let alwaysShowSearchSuggestionsView = browserViewControllerState?.searchScreenState.showSearchSugestionsView ?? false
```

Comment:
Line `1900` replaced `let isPrivateTab = tabManager.selectedTab?.isPrivate ?? false`. Since we are using the `showSearchSuggestionsView` value instead as a part of Redux, I renamed this boolean to make more sense. We want to `alwaysShowSearchSuggestionsView` when in Normal Mode, but please let me know if there is a better way to make this logic more clear.